docs/en/sql-reference/functions/string-search-functions.md

```diff
@@ -590,6 +590,10 @@ Result:
 └───────────────────────────────┘
 ```
 
+## countMatchesCaseInsensitive
```

Comment:
Should be added to `aspell-dict.txt` to whitelist (and unblock the tests).

Reply:
Done, thanks.